src/citrine/informatics/descriptors.py

```diff
@@ -19,7 +18,8 @@ def get_type(cls, data) -> Type[Serializable]:
         return {
             "Real": RealDescriptor,
             "Inorganic": InorganicDescriptor,
-            "Categorical": CategoricalDescriptor
+            "Categorical": CategoricalDescriptor,
+            "Organic": MolecularStructureDescriptor
```

Comment:
Adding a hanging comma onto the end of all multiline lists will help reduce spurious lines from your PRs. Specifically, here, you didn't really edit line 21, yet it shows up in the PR because you needed to add a comma so that you could add line 22. If you add a comma to the end of line 22 now, you'll avoid issues like this in the future

Reply:
Good call, thanks!